### PR TITLE
net/freeradius: Allow `&` as a password character

### LINKED ISSUE #4115: os-freeradius: Allow `&` in users' password
**Important notices**
Before you add a new report, we ask you kindly to acknowledge the following:

- [x] I have read the contributing guide lines at https://github.com/opnsense/plugins/blob/master/CONTRIBUTING.md
- [x] I have searched the existing issues, open and closed, and I'm convinced that mine is new.
- [x] When the request is meant for an existing plugin, I've added its name to the title.

**Is your feature request related to a problem? Please describe.**
`&` is a common character in password patterns. 

**Describe the solution you'd like**
I want to be able to input `&` in the password field

**Additional context**
image

### PR DESCRIPTION
**Pull request by captainko**
Fixes #4115 

Allow `&` to in password

### CODE REVIEW

File: net/freeradius/src/opnsense/mvc/app/controllers/OPNsense/Freeradius/forms/dialogEditFreeRADIUSUser.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
         <id>user.username</id>
         <label>Username</label>
         <type>text</type>
-        <help>Set the unique username for the user. Allowed characters are 0-9, a-z, A-Z, and ._-@/:</help>
+        <help>Set the unique username for the user. Allowed characters are 0-9, a-z, A-Z, and ._-@/:&amp;</help>
```
Comment (fichtner) [inline net/freeradius/src/opnsense/mvc/app/controllers/OPNsense/Freeradius/forms/dialogEditFreeRADIUSUser.xml:12]: This is a case were wrapping this in CDATA is a better approach to use the literal character.

File: net/freeradius/src/opnsense/mvc/app/models/OPNsense/Freeradius/User.xml [hunk 2]
```diff
@@ -15,7 +15,7 @@
                 </username>
                 <password type="TextField">
                     <Required>Y</Required>
-                    <mask>/^([0-9a-zA-Z._\-\!\$\%\/\(\)\+\#\=\{\}:]){1,128}$/u</mask>
+                    <mask>/^([0-9a-zA-Z._\-\!\$\%\/\(\)\+\#\=\{\}:&amp;]){1,128}$/u</mask>
```
Comment (fichtner) [inline net/freeradius/src/opnsense/mvc/app/models/OPNsense/Freeradius/User.xml:18]: Same here.

File: net/freeradius/src/opnsense/mvc/app/controllers/OPNsense/Freeradius/forms/dialogEditFreeRADIUSUser.xml (hunk 1 above)
Comment (captainko) [reply net/freeradius/src/opnsense/mvc/app/controllers/OPNsense/Freeradius/forms/dialogEditFreeRADIUSUser.xml]: Thanks. I have updated it.

**Comment by captainko** [comment]
Please kindly check again

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**